### PR TITLE
Create security-and-privacy.md

### PR DESCRIPTION
**Pull request by nigelmegitt**
Needed to file the TAG review request issue.

### CODE REVIEW

File: misc/security-and-privacy.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+This questionnaire has [moved](https://w3ctag.github.io/security-questionnaire/).
```
Comment (palemieux) [inline misc/security-and-privacy.md:3]: What does "move" mean? From where to where?

File: misc/security-and-privacy.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+# Security and Privacy Self-review for IMSC-HRM
+
+This is an edited version of the security and privacy questionnaire template at
+https://raw.githubusercontent.com/w3ctag/security-questionnaire/main/questionnaire.markdown as linked from
```
Comment (palemieux) [inline misc/security-and-privacy.md:8]:
```suggestion
https://w3ctag.github.io/security-questionnaire/ as linked from
```

File: misc/security-and-privacy.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+[Self-Review Questionnaire: Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/)
+
+> 01.  What information might this feature expose to Web sites or other parties,
+>      and for what purposes is that exposure necessary?
+
+None. The IMSC-HRM is a static-analysis algorithm of IMSC documents.
+Executing the algorithm requires no requests to any origins.
```
Comment (palemieux) [inline misc/security-and-privacy.md:15]: Is it worth explicitly noting that "This specification does not define any protocol or interface for obtaining such a resource, and it does not define any interface for exposing the results of the analysis." as mentioned at https://github.com/w3c/imsc-hrm/issues/19#issue-1073404996

**Review by palemieux — APPROVED** [review]
Suggest adding links to the PING and security reviews: https://github.com/w3cping/privacy-request/issues/65 and https://github.com/w3c/security-request/issues/18

File: misc/security-and-privacy.md (hunk 1 above)
Comment (nigelmegitt) [reply misc/security-and-privacy.md:3]: Oh I don't know, that's in the template I copied!

File: misc/security-and-privacy.md (hunk 2 above)
Comment (nigelmegitt) [reply misc/security-and-privacy.md:8]: I think the better link to use is https://github.com/w3ctag/security-questionnaire/blob/main/questionnaire.markdown if anything, but I'm not sure what's wrong with the original one?

File: misc/security-and-privacy.md (hunk 3 above)
Comment (nigelmegitt) [reply misc/security-and-privacy.md:15]: Yes will do.

File: misc/security-and-privacy.md (hunk 1 above)
Comment (nigelmegitt) [reply misc/security-and-privacy.md]: Removed in e971621.

File: misc/security-and-privacy.md (hunk 3 above)
Comment (nigelmegitt) [reply misc/security-and-privacy.md:18]: Done in e971621.

**Pull request merged**